src/cljs/athens/self_hosted/presence/subs.cljs

```diff
@@ -47,6 +47,15 @@
     (:username current-user)))
 
 
+(rf/reg-sub
+  :presence/user-page
+  :<- [:presence/current-username]
```

Comment:
I think what should happen here is `:presence/current-username` returns `You` when in PKM. That way other features get this behaviour automatically.

But other features might already be checking if PKM or CKM as well so this refactor might be a bit harder. I think I'll tackle it in another PR.